src/main/java/com/hankcs/algorithm/AhoCorasickDoubleArrayTrie.java

```diff
@@ -114,6 +114,32 @@ public void parseText(String text, IHit<V> processor)
      * @param text The text
      * @param processor A processor which handles the output
      */
+    public void parseText(String text, IHitCancellable<V> processor)
```

Comment:
Could `parseText(String, IHit<V>)` delegate to this one, with `processor` set to an object for which `hit` always returns `false`?